hack/test-end-to-end.sh

```diff
@@ -168,25 +171,38 @@ echo "[INFO] Using images:              ${USE_IMAGES}"
 
 # Start All-in-one server and wait for health
 # Specify the scheme and port for the listen address, but let the IP auto-discover.	Set --public-master to localhost, for a stable link to the console.
+echo "[INFO] Minting certificates for the OpenShift server"
+# find the same IP that openshift start will bind to.  This allows access from pods that have to talk back to master
+ALL_IP_ADDRESSES=`ifconfig | grep "inet " | awk '{print $2}'`
+SERVER_HOSTNAME_LIST="${PUBLIC_MASTER_HOST},localhost"
+while read -r IP_ADDRESS
+do
+	SERVER_HOSTNAME_LIST="${SERVER_HOSTNAME_LIST},${IP_ADDRESS}"
+done <<< "${ALL_IP_ADDRESSES}"
+
+openshift certs mint-all-certs --overwrite=false --cert-dir="${CERT_DIR}" --hostnames="${SERVER_HOSTNAME_LIST}" --nodes="127.0.0.1"
+echo "[INFO] Create .kubeconfig files for clients"
+CLIENTS=( "admin" "openshift-client" "openshift-deployer" "kube-client" "node-127.0.0.1")
+for USER in "${CLIENTS[@]}"
+do
+	openshift certs create-kubeconfig --master="${MASTER_ADDR}" --certificate-authority="${CERT_DIR}/ca/cert.crt" --client-certificate="${CERT_DIR}/${USER}/cert.crt" --client-key="${CERT_DIR}/${USER}/key.key" --kubeconfig="${CERT_DIR}/${USER}/.kubeconfig" --user="${USER}"
+done
+
+
 echo "[INFO] Starting OpenShift server"
 sudo env "PATH=${PATH}" OPENSHIFT_ON_PANIC=crash openshift start \
-     --listen="${API_SCHEME}://0.0.0.0:${API_PORT}" --public-master="${API_SCHEME}://${PUBLIC_MASTER_HOST}" \
+     --listen="${API_SCHEME}://0.0.0.0:${API_PORT}"  --master="${MASTER_ADDR}" --public-master="${API_SCHEME}://${PUBLIC_MASTER_HOST}" \
      --hostname="127.0.0.1" --volume-dir="${VOLUME_DIR}" \
      --etcd-dir="${ETCD_DATA_DIR}" --cert-dir="${CERT_DIR}" --loglevel=4 \
-     --images="${USE_IMAGES}" \
+     --images="${USE_IMAGES}" --mint-certs=false\
```

Comment:
`--create-certs` - also.... why would you have to create them if they were already created?